content/soarian/dstu2/general-clinical/condition.md

```diff
@@ -0,0 +1,83 @@
+
+
+## Terminology Bindings  
+
+<%= terminology_table(:soarian_condition, :dstu2) %>
+
+## Extensions
+The content of these extensions must be visible to the user, and recorded with any storage or transmit activity along with the query/response date/time, as it represents the best representation of what clinicians recorded for the patient at the time of query.  
+
+
+| Name | Condition | Description |
+| --- | --- | --- |
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|Problem list has no qualifying entries|**No relevant problems exist**|
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|Problem list contains qualifying entries|The following labels and values from the list will be returned in HTML, if valued: **Charted Problem Name, Assigned Date, Location, Status, Event, Day Part, Body Site, Frequency, Time Pattern, Intermittent, Rank, Severity, Body Location, Onset Date, Comment, Certainty, Clinical Care Category**|
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|Problem list contains qualifying entries|Informational message that must be shown to the user: **This is the most current, known Problem list and may not reflect the state of problems known at a previous date.**|
+
+
+**CAUTION:** The detail that is in the narrative is not fully expressed in the structured portion of the FHIR response: The https://www.hl7.org/fhir/narrative-definitions.html#Narrative.status entry in the narrative is valued as "Additional" indicating that more information is provided in the narrative than is provided in the structure. Therefore, if the Application is using the structured entries for any purpose, care must be taken to ensure the clinical content/context of the narrative is preserved.  The structured component of the FHIR Condition resource response will primarily contain the code, code system, and code description for any coded problems in the list returned in the Narrative. This portion of the response may not be relevant to the user and should not be displayed. 
+
+
+## Search           
+
+
+### Parameters      
+
+No Condition resource specific search parameters are supported. For an unambiguous patient-facing problem list, the current problem list is always returned; date date/range parameters are not used for this resource.
+
+|Name |Required? | Type | Description
+| --- | --- | --- | --- |
+| patientId | Y | reference | The patient identifier provided in a pre-requisite authorization step. |
+
+### Header
+To successfully GET Condition, the following headers must be provided.  
+
+    Accept-Type: application/json+fhir
```

Comment:
Similar to diagnostic report, just use the headers method w/no params to get this output in the correct format